print env URL during some CLI commands. stop printing "Using config file: __"

emily-potyraj opened the pull request:
1. Print info about accessing env URL during the following commands:  
- rad env init azure
- rad env switch
- rad deploy
2. Stop printing `Using config file: __` every time we read the config file via initConfig. Closes: radius-project/radius#254, radius-project/core-team#796
3. During `rad deploy`, print which env is being used. 


Not sure whether we prefer to add error handling inside GenerateEnvUrl (in utils.go) or in the other files where GenerateEnvUrl is called - or both. 
_____

Sorry for any confusion about SUPPORT.md being in this PR. Until that EOL change gets merged, git makes it a mandatory change for me.

Commit:
stage 1 of change: envInitAzure, deploy

Commit:
stage 1 of change: envInitAzure, deploy

Commit:
nuisance EOL issue for SUPPORT.md preventing me from rebasing.
can't even stash it.

Commit:
add URL during envSwitch. add layer to use kind to select URL builder function.

Commit:
add URL during envSwitch. add layer to use kind to select URL builder function.

rynowak reviewed (commented):
Left some minor feedback, overall looks to be on track 👍

Commit:
tidy up envUrl generation
- only support kind=azure for now
- make tenantID a variable instead of hardcoding to Microsoft internal
- formatting improvements

emily-potyraj committed:
Update pkg/rad/azure/env_url.go

Co-authored-by: Aaron Crawfis <Aaron.Crawfis@microsoft.com>

rynowak reviewed (approved):
Awesome!
